public/index.html

```diff
@@ -69,6 +69,20 @@
       type="font/woff2"
       crossorigin="anonymous"
     />
+    <link
+      rel="preload"
+      href="ZhiMangXing-Regular.woff2"
+      as="font"
+      type="font/woff2"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="preload"
+      href="ZCOOLQingKeHuangYou-Regular.woff2"
+      as="font"
+      type="font/woff2"
+      crossorigin="anonymous"
+    />
```

Comment:
Let's remove this. This will preload those ~5MB fonts for every user, which we don't want.